src/histolab/util.py

```diff
@@ -100,24 +100,24 @@ def random_choice_true_mask2d(binary_mask: np.ndarray) -> Tuple[int, int]:
     Returns
     -------
     Tuple[int, int]
-        Random pair of indices where the ``binary_mask`` is True.
+        Random pair of indices (column, row) where the ``binary_mask`` is True.
     """
-    x = np.random.choice(np.where(binary_mask)[0])
-    y = np.random.choice(np.where(binary_mask[x])[0])
+    true_y, true_x = np.where(binary_mask)
```

Comment:
IMHO this should still be `x,y` the `true` prefix is weird to me. The fact the it calculates the pair of indices where the mask is true is specified well enough in the docstring, so this variables should maintain the old names